src/query/procedure/mg_procedure_impl.cpp

```diff
@@ -1966,11 +2006,21 @@ mgp_error mgp_edge_get_type(mgp_edge *e, mgp_edge_type *result) {
 }
 
 mgp_error mgp_edge_get_from(mgp_edge *e, mgp_vertex **result) {
+  if (e->from.graph->ctx && e->from.graph->ctx->auth_checker &&
+      !e->from.graph->ctx->auth_checker->Accept(*e->from.graph->ctx->db_accessor, e->from.impl, e->from.graph->view)) {
+    return mgp_error::MGP_ERROR_NO_ERROR;
+  }
+
   *result = &e->from;
   return mgp_error::MGP_ERROR_NO_ERROR;
```

Comment:
If you keep this check (see the discussion @Josipmrden started), then use `WrapExceptions` as the `Accept` function is not `noexcept`.

Reply:
This one is going out, mgp_edges for which the user can't see both vertices are filtered out. So, this check is redundant.